backend/src/main/resources/application.yaml

```
@@ -19,6 +19,8 @@ spring:
   jackson:
     serialization:
       FAIL_ON_EMPTY_BEANS: false
+  session:
+    store-type: jdbc
```

The session will need to time out after 15 minutes. From [the Spring docs](https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html#spring.session.timeout), I think this is the correct syntax:

```suggestion
  session:
    store-type: jdbc
    timeout: 900
```

---

Good suggestion, thank you! Updated.